frontends/web/src/routes/settings/settings.tsx

```diff
@@ -367,7 +367,7 @@ class Settings extends Component<Props, State> {
                                                         </div>
                                                         <SettingsButton
                                                             onClick={this.showProxyDialog}
-                                                            optionalText={config.backend.proxy.useProxy ? 'Enabled' : 'Disabled'}>
+                                                            optionalText={config.backend.proxy.useProxy ? t('settings.expert.proxyEnabled') : t('settings.expert.proxyDisabled')}>
```

Comment:
Already got generic text for Enabled/Disabled, please use as in this commit in the open PR: https://github.com/digitalbitbox/bitbox-wallet-app/pull/758/commits/dc881479fff705334c603b69ab50c271a3f312cf

Reply:
Sorry, what do you want me to do? use `generic.enabled.false` and `generic.enabled.true`?

Reply:
You would have to do this: `optionalText={t('generic.enabled', {context: config.backend.proxy.useProxy.toString()})}`

Reply:
When using `optionalText={t('generic.enabled', {context: config.backend.proxy.useProxy.toString()})}>`It does not show the "Enabled or disabled" label at all.

Reply:
That's because that functionality is introduced in a still open PR which I linked to in the first comment. Either build on top of that, or just copy the relevant `generic` section into your `app.json` file. Then it will work

Reply:
I see, thx